### PR TITLE
changed main logo to svg, made slightly larger

### LINKED ISSUE #43: Replace main logo on website with SVG
The SVG is on our Google Drive somewhere. Let's change the main logo to an SVG to cut down on the size of the page.

### PR DESCRIPTION
**Pull request by troynguyen8**
Changed main logo in first section from png to svg. Used this svg:

https://drive.google.com/open?id=1ngygesv0pYIJiHQgVxpR_eYC1vA9Sfyu

Edited SVG to remove unnecessary metadata, made height and width in pixels instead of inches in svg as well.

Validated in Chrome, Edge, and IE11

Closes #43

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
     <main class="row">
       <section class="girlscode fullwidth col-12" data-parallax="scroll" data-image-src="./img/background.jpg">
-        <img src="/img/girlscodelincoln.png" alt="Girls Code Lincoln" />
+        <img src="/img/girlscodelincoln.svg" alt="Girls Code Lincoln" />
```
Comment (justinjstark) [inline src/index.html]: Why did you change all the URLs relative to root (/img) instead of (img)? It broke images when browsing locally.

Comment (troynguyen8) [reply src/index.html]: I don't have a great reason, I was just trying to standardize it. I can change them all back to relative if you'd like.

By the way, I've been running the source folder on local host (refer to README) to make sure changes look exactly as they would when deployed on local machines
